events/iot.go

```diff
@@ -0,0 +1,22 @@
+package events
+
+// IoTCustomAuthorizerRequest contains data coming in to a custom IoT device gateway authorizer function.
+type IoTCustomAuthorizerRequest struct {
+	AuthorizationToken string `json:"token"`
+}
+
+// IoTCustomAuthorizerResponse represents the expected format of an IoT device gateway authorization response.
+type IoTCustomAuthorizerResponse struct {
+	IsAuthenticated          bool                   `json:"isAuthenticated"`
+	PrincipalID              string                 `json:"principalId"`
+	DisconnectAfterInSeconds int32                  `json:"disconnectAfterInSeconds"`
+	RefreshAfterInSeconds    int32                  `json:"refreshAfterInSeconds"`
+	PolicyDocuments          []string               `json:"policyDocuments"`
+	Context                  map[string]interface{} `json:"context,omitempty"`
```

Comment:
I don't see a context field in the documentation. Is there another reference that contains this?

from https://docs.aws.amazon.com/iot/latest/apireference/API_TestInvokeAuthorizer.html#API_TestInvokeAuthorizer_ResponseElements
```
{
   "disconnectAfterInSeconds": number,
   "isAuthenticated": boolean,
   "policyDocuments": [ "string" ],
   "principalId": "string",
   "refreshAfterInSeconds": number
}
```